qtpy/__init__.py

```diff
@@ -100,12 +100,21 @@ class PythonQtWarning(Warning):
 # Names of the expected PySide2 api
 PYSIDE2_API = ['pyside2']
 
+VALID_APIS = PYQT5_API + PYQT4_API + PYSIDE_API + PYSIDE2_API
+
+# Record if an explicit version of Qt was requested, so we can report this if
+# an error occurs.
+_EXPLICIT_API = None
+if QT_API in os.environ:
+    _EXPLICIT_API = os.environ[QT_API]
```

Comment:
Are there any other variables names that could have been used? or is this exclusive?

Reply:
I'm don't quite understand the question, could you rephrase?

Reply:
@Erotemic what I wat to say if QT Docs only talk about `QT_API` as an environment variable, or are there other variants?

Reply:
I think `QT_API` was defined by the Python community (probably by Matplotlib or IPython).

Reply:
Hmm ok :-), then it should be fine